lambdas/data-migration2/tests/test-executions.js

```diff
@@ -442,12 +442,23 @@ test.serial('migrateExecutions processes multiple executions', async (t) => {
     executionsModel.create(newExecution),
     executionsModel.create(newExecution2),
   ]);
+
+  console.log('newExecution.arn', newExecution.arn);
```

Comment:
Nit: Should we keep these?  They could be useful in debugging, but 50/50 on if they're just logspam as debugging is likely needed if the test fails.     Up to you.

Reply:
dfda99d54ee1397ee7ab8be22337ffa0a85dd651